wpilib/wpilib/doublesolenoid.py

```diff
@@ -111,9 +111,9 @@ def __init__(self, *args, **kwargs):
                                             self.forwardHandle, self.reverseHandle)
 
     def free(self):
-        """Mark the solenoid as freed."""
-        LiveWindow.removeComponent(self)
-        
+        """
+        Mark the solenoid as freed.
+        """
```

Comment:
Uh, why did you reformat this docstring?

Reply:
it annoyed me

Reply:
well, this is annoying me now 😜

(see PEP 257)

Reply:
ugh, I suppose next you're going to tell me there's a convention against a comma following the last item in an array literal

Reply:
I actually prefer adding a comma to the end of an array literal. 👍